### DISCUSSION TITLE
Update / Auto-Update

### DISCUSSION THREAD

**Discussion by Marco-Computer** [geimist/synOCR#40] [Q&A] [answered]
Hallo, auf einer Synology DS720+ läuft aktuell synocr 1.4.1. Stabil und Super Schnell, mit automatischer Ordnererkennung wenn etwas in den Ordner reingelegt wird / dorthin gescannt wird. Jetzt meldet das gui auf der NAS das 1.4.5 verfügbar ist.  Die spk Datei habe ich heruntergeladen. Gibt es eine Anleitung dazu wie ich die NAS ohne das die Einstellungen verloren werden updaten kann ? Habe erfolglos im Wiki gesucht, steht ja auch noch im Aufbau. Ein Tipp oder kurze Anleitung würde mir helfen. Danke. 

Vielleicht gibt es ja auch als Idee einen Update Knopf, der auf eine Anleitung mündet, oder sogar dann selber alles macht ?

**Comment by geimist** [comment, marked as answer]
Einfach im Paketzentrum drüberinstallieren. Die Einstellungen bleiben dabei erhalten. Eine Backup der Einstellungen via HyperBackup schadet allerdings nie 😊

**Reply by Marco-Computer** [reply]
Guten Morgen, danke für die schnelle Rückmeldung. Hyperbackup läuft sowieso :-) Habe gerade über das  Paketzentrum oben rechts manuelle Installation die spk Datei eingelesen, und natürlich hat es direkt geklappt. Danke dafür. Auch für die gute Arbeit !!!
